lib/post_destroyer.rb

```diff
@@ -305,7 +305,6 @@ def mark_topic_changed
   def make_previous_post_the_last_one
     last_post =
       Post
-        .where("topic_id = ? and id <> ?", @post.topic_id, @post.id)
```

Comment:
O look this is the same line of code as two lines down :grin:

Reply:
This is called "extra where" it's actually a valid SQL feature, look it up :P

Reply:
There's also `limit(1).first`, but `#first` already adds a `limit(1)`. 🙂